app_notifier/src/app_notifier/mail_notifier_plugin.py

```diff
@@ -17,6 +17,16 @@ def __init__(self):
 
     def app_manager_start_plugin(self, app, ctx, plugin_args):
         self.start_time = rospy.Time.now()
+        # Set mail title first for extensibility
```

Comment:
What do you mean by 'extensibility'?

Reply:
I updated comment([b08f8b7](https://github.com/knorth55/app_manager_utils/pull/55/commits/b08f8b7b80765e758bbb2fdcd877f08cdcd597db))
I clarified more.